src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java

```diff
@@ -49,4 +49,10 @@ public class AuctionHouseConfig {
     )
     @ConfigEditorKeybind(defaultKey = Keyboard.KEY_NONE)
     public int copyUnderbidKeybind = Keyboard.KEY_NONE;
+
+    @Expose
+    @ConfigOption(name = "Price Website", desc = "Adds a button to the Auction House that will open the item page in §ccoflnet.com§7.")
```

Comment:
To be consistent between both descriptions
```suggestion
    @ConfigOption(name = "Price Website", desc = "Adds a button to the Auction House that will open the item page in §csky.coflnet.com§7.")
```